codecov.yml

```diff
@@ -5,5 +5,5 @@ coverage:
   status:
     project:
       default:
-        threshold: 5%
+        threshold: 20%
```

Comment:
Do we care about the threshold. I think we should put this at 100% and only care about the overall value